fr/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md

```diff
@@ -0,0 +1,350 @@
+---
+id: operatingsystems-linux-ssh
+title: Linux SSH
+---
+
+## Vue d'ensemble
+
+Linux est une famille de systèmes d'exploitation à source ouverte de type Unix basée sur le noyau Linux, un noyau de système d'exploitation publié pour la première fois le 17 septembre 1991 par Linus Torvalds. 
+
+## Contenu du pack de supervision
+Ce Plugin-Pack est compatible avec n'importe quelle distribution Linux avec un daemon SSH installé :
+### Objets supervisés
+
+* Serveur
+* Centos 
+* Redhat
+* Debian
+* Ubuntu
+* Fedora
+* Suse
+
+### Métriques collectées
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Cpu-->
+
+| Metric name                        | Description                                    |
+| :--------------------------------- | :--------------------------------------------- |
+| cpu.utilization.percentage         | CPU utilization. Unit: %                       |
+| core.cpu.utilization.percentage    | CPU utilization by core. Unit: %               |
+
+
+<!--Cpu Detailled-->
+
+| Metric name           | Description                             |
+| :-------------------- | :-------------------------------------- |
+| cpu                   | Average usage for each CPUs. Unit : %   |
+| total_cpu             | Average total CPUs. Unit : %            |
```

Comment:
Cpu detailed il y a que ces métriques ?

Reply:
Comme vu cela été mis à jour.
* Per CPU :

| Metric name     | Description                           |
| :---------------| :------------------------------------ |
| cpu_user        | Average user for CPU. Unit : %       |
| cpu_nice		  | Average nice for CPU. Unit : %       |
| cpu_system	  | Average system for CPU. Unit : %     |
| cpu_idle		  | Average idle for CPU. Unit : %       |
| cpu_wait		  | Average wait for CPU. Unit : %       |
| cpu_interrupt	  | Average interrupt for CPU. Uni.t : %  |
| cpu_softirq	  | Average softirq for CPU. Unit : %    |
| cpu_steal       | Average steal for CPU. Unit : %      |
| cpu_guest       | Average guest for CPU. Unit : %      |
| cpu_guestnice   | Average guest nice for CPU. Unit : % |


* Total :
 
| Metric name              | Description                             |
| :----------------------- | :-------------------------------------- |
| total_cpu_guest_avg      | Total average guest CPU. Unit : %      |
| total_cpu_guestnice_avg  | Total average guest nice CPU. Unit : % |
| total_cpu_idle_avg	   | Total average idle CPU. Unit : %       |
| total_cpu_interrupt_avg  | Total average interrupt CPU. Unit : %  |
| total_cpu_nice_avg       | Total average nice CPU. Unit : %       |
| total_cpu_softirq_avg	   | Total average softirq CPU. Unit : %    |
| total_cpu_steal_avg      | Total average steal CPU. Unit : %      |
| total_cpu_system_avg     | Total average system CPU. Unit : %     |
| total_cpu_user_avg 	   | Total average user CPU. Unit : %       |
| total_cpu_wait_avg       | Total average wait CPU. Unit : %       |